src/cascadia/WinRTUtils/pch.h

```diff
@@ -0,0 +1,30 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+// pch.h
+// Header for platform projection include files
+//
+
+#pragma once
+
+#define WIN32_LEAN_AND_MEAN
+
+#include <LibraryIncludes.h>
+// This is inexplicable, but for whatever reason, cppwinrt conflicts with the
+//      SDK definition of this function, so the only fix is to undef it.
+// from WinBase.h
+// Windows::UI::Xaml::Media::Animation::IStoryboard::GetCurrentTime
+#ifdef GetCurrentTime
+#undef GetCurrentTime
+#endif
+
+#include <wil/cppwinrt.h>

```

Comment:
Should this one be in `LibraryIncludes.h`? Or is it too sensitive to the include order?

Reply:
I believe this one has a sensitive include order.. but i am not sure.
We may also not want to include it in LibraryIncludes for _all conhost stuff_

Reply:
OK

Reply:
conhost, it might actually be warranted for the TSF stuff that is COMish. but its fine.